predicate/README.md

```diff
@@ -83,11 +82,11 @@ metadata:
   name: access
 spec:
   allow:
-    node: (((node.login == user.name) && (!(user.name == "root"))) || equals(user.traits["team"],
-      ["admins"]))
+    accessnode: (((access_node.login == user.name) && (!(user.name == "root"))) ||
```

Comment:
here is a bit of a contradiction, we have both `accessnode` and `access_node` what is confusing. Let's use `access_node` everywhere, this matches our current RBAC style too (using underscores).

Reply:
Done in 10a57714a7670069748a28d4c30ca7f3081cae60.